### DISCUSSION TITLE
Where are release notes for v12 releases?

### DISCUSSION THREAD

**Discussion by askkaz** [vercel/next.js#43261] [Help]
I've noticed that recent v12 releases (v12.3.4, v12.3.3, v12.3.2, v12.2.6) don't have any release notes/changelog in the usual place. Are they somewhere else? https://github.com/vercel/next.js/tags

**Comment by icyJoseph** [comment]
Ah, interesting, perhaps they are releasing bug/experimental fixes for some, strange that they don't use canary tags for these though. Here's what's changed:

https://github.com/vercel/next.js/compare/v12.3.2...v12.3.4
